### DISCUSSION TITLE
How Remove Static Generated Page without build again

### DISCUSSION THREAD

**Discussion by MortezaNajafian** [vercel/next.js#20831] [Help] [answered]
Hello to all friends
How to delete a page created statically by the GetStaticProps without running the npm run build ?

**Comment by jamesmosier** [comment, marked as answer]
Hi @nilootech. Have you tried the `revalidate` option yet? https://nextjs.org/docs/basic-features/data-fetching#getstaticprops-static-generation

Also this Discussion might be of some help: https://github.com/vercel/next.js/discussions/18967
